docs/fava.rst

```diff
@@ -23,33 +39,28 @@ The important parts of a rule plugin:
   * a name, identifier
   * a condition, a Boolean expression of the data from the parsers
   * a set of pydata, a set of key/value pairs giving additional,
-     machine specific details about the rule-hit using data from the parsers
-     The output of a rule plugin is either nothing (None) indicating that the condition evaluated to false, or a combination of the identifiers name and the pydata.
-
-For example:
+    machine specific details about the rule-hit using data from the parsers
+    The output of a rule plugin is either nothing (None) indicating that the condition evaluated to false,
+    or a combination of the identifiers name and the pydata.
 
+For example::
 
-```yaml
----
-rule:
-  name: "UNAME_TEST"
-  pydata:
-    kernel: "{{ Uname.kernel }}"
-  when: Uname.fixed_by('2.6.32-431.11.2.el6', introduced_in='2.6.32-431.el6')
-```
-
+  ---
```

Comment:
You can use the following for code blocks if you want syntax highlighting:
```
.. code block:: yaml
```
See https://github.com/RedHatInsights/insights-core/blob/master/docs/customtut_parsers.rst for some examples.  Also see Sphinx reference: http://www.sphinx-doc.org/en/stable/markup/code.html